src/components/EmojiPicker/EmojiPickerMenu/BaseEmojiPickerMenu.tsx

```diff
@@ -139,7 +112,7 @@ function BaseEmojiPickerMenu({headerEmojis, scrollToHeader, isFiltered, listWrap
                     ListEmptyComponent={ListEmptyComponent}
                     alwaysBounceVertical={alwaysBounceVertical}
                     estimatedItemSize={CONST.EMOJI_PICKER_ITEM_HEIGHT}
-                    estimatedListSize={{height: flattenListWrapperStyle.height, width: listWidth}}
+                    estimatedListSize={{height: flattenListWrapperStyle.height as number, width: listWidth}}
```

Comment:
How about a default value here?
```suggestion
                    estimatedListSize={{height: flattenListWrapperStyle.height ?? 0, width: listWidth}}
```